### ISSUE TITLE
Unable to Juju ssh from a user with write access to the model

### ISSUE THREAD

**Issue by anastasiamac** [juju/docs#1417]
According to https://bugs.launchpad.net/juju/+bug/1629141, we need to clarify in the docs that 'juju add-ssh-key' run by the admin user that created the model gives the other user access via ssh.

**Comment by pmatulis** [comment]
@anastasiamac So only the admin user can add an SSH key to a model? I believe previously a non-admin user, who was provided model access, would internally make use of some admin/juju SSH key.

**Comment by evilnick** [comment]
see also https://bugs.launchpad.net/juju/+bug/1629835

**Comment by chuckbutler** [comment]
@anastasiamac @pmatulis  - so to be very specific what worked in our (~containers) case:

we imported the juju-client-public-key that's located in `$HOME/.local/share/juju/ssh` and we were able to juju ssh into the unit. If we imported say our public key, it didn't work as expected.

I feel this should also be added to that outline referenced above, otherwise its misleading... and juju doesn't do a great job of reading my mind that it should be using my personal public key vs the public key in `$JUJU_DATA`.
